### ISSUE TITLE
About item-based functions

### ISSUE THREAD

**Issue by Rigaln** [TeamFightTacticsBots/Alune#60]
I'm using ChatGPT to translate this because I can't speak English. I'm sorry if it's difficult to read or understand.

I've been gratefully using it since yesterday.

Would it be possible to implement the item-related features mentioned in the roadmap sooner? The reason is that when items are scattered around the playing field, they stand out a lot, and I feel that it could make others suspicious.

**Comment by Jokiblack** [comment]
It is not yet implemented, because I think, if I'm wrong, correct me:
The bot is not able to detect the champions on the board, nor on the bench
The images in the mobile TFT that are seen on the board are of very poor quality, which makes it difficult for the bot to correctly place the items
So we would have to find a way to be able to identify all the champions available in this set, and randomly place items on anyone, but the quality of the champions' TFT is very bad, and on top of that they move
If I'm wrong, correct me

**Comment by Jokiblack** [comment]
![image](https://github.com/user-attachments/assets/fc41c526-c2f2-473d-99a6-04c264be72e3)
It's still a crazy idea, I don't know if it's possible to make the bot drag the item randomly, to a champion for example, every 1 minute try to drag an item to any of those holes, which are always filled because the tft automatically positions the champions, if you drag it and leave it there, it's random but items don't accumulate, sometimes it will happen that it's an anvil, or a change, the change and the anvil that omits them and doesn't drag the rest of the items, if randomly to any champion

**Comment by Jokiblack** [comment]
and with that we can make him also pick up the balls, and if it works, add that every 10 minutes for example he sells the whole bench

**Comment by Rigaln** [comment]
I think the idea is good.
Personally, the most important thing for me is not to raise suspicion, so I would be happy if collecting the balls on the field is prioritized.

**Comment by Jokiblack** [comment]
> I think the idea is good. Personally, the most important thing for me is not to raise suspicion, so I would be happy if collecting the balls on the field is prioritized.

This was already intended to be implemented, but there was a problem, in that if the bench or the items are full, it does not collect more and sends many requests without being able to fulfill them and could generate lag and saturate the bot.

**Comment by akshualy** [comment]
Yes, assigning the items randomly to units on the board would be the first step.
In the previous PC version of this bot I already prepared a method to detect unit positions on the board, it would be possible to apply the same strategy here.

**Comment by Jokiblack** [comment]
> Yes, assigning the items randomly to units on the board would be the first step. In the previous PC version of this bot I already prepared a method to [detect unit positions on the board](https://github.com/Kyrluckechuck/TFT-Bot/blob/2edaae227f6129dca22b75da3c35a4df48cebb3c/tft_bot/helpers/screen_helpers.py#L317-L358), it would be possible to apply the same strategy here.

I guess it won't be as easy as copy and paste, or telling chatgpt to implement it haha

**Comment by Jokiblack** [comment]
I tried to adapt it with the help of chatgpt but I get an error with the win32 module

**Comment by akshualy** [comment]
I'll implement the new game mode today and tomorrow, after that I can take another look at assigning items.
